tests/dynamic_module/MiaTestModule.cpp

```diff
@@ -0,0 +1,24 @@
+#include <Generator.hpp>
+
+#include <cppast/visitor.hpp>
+
+class TestModule
+	: public mia::GeneratorModule
+{
+	const char* getVersion() const override
```

Comment:
Maybe add some utility class to Generator.hpp to avoid implementing this method since the body should always be the same.

Reply:
Added `GeneratorModuleBase` which does basically this.